iset.mm

```diff
@@ -64617,6 +64617,16 @@ elements or fails to hold (is equal to ` (/) ` ) for some element.
     BRVETUACZATUBZEFVCVNVOVEUCUIVEADVPEVCVDUDUEUFVMATBUGUHVGUJVHABUKULUMBAVHUNU
     OUPUTUQVCVKBURVDBAUSVAVB $.
 
+  $( Disjoint union version of one plus one equals two.  (Contributed by Jim
+     Kingdon, 1-Jul-2022.) $)
+  dju1p1e2 $p |- ( 1o |_| 1o ) ~~ 2o $=
```

Comment:
Nice theorem. Plus, the funny face that it represents in ascii art proves that " |_| " was the best choice for disjoint union.